packages/runtime/src/widgets/Row.tsx

```diff
@@ -26,11 +28,19 @@ export const Row: React.FC<FlexboxProps> = (props) => {
   const renderedChildren = useMemo(() => {
     return children ? EnsembleRuntime.render(children) : null;
   }, [children]);
-
+  const action = useEnsembleAction(props?.onTap);
```

Comment:
Destructure this from the props

Reply:
got it @evshi, but i have one question and it is solely for reason of curiosity. If you can answer it please. 
Is there any performance or code quality reason to destructure first and then use vs directly using a prop.

Reply:
Yes, in this case if you destructure it first, then it won't be passed into the register bindings so saves extra processing